### DISCUSSION TITLE
Ability to replace/remove all prefix from files

### DISCUSSION THREAD

**Discussion by anwarulislam** [tailwindlabs/tailwindcss#9152] [Ideas]
Hi, this would be interesting if we could replace/remove all prefixes after changing the prefix from our tailwind config. This can be helpful for those who started using tailwind with other frameworks using prefixes not to overlap with others and later refactored their codebase. transformed all code to tailwind. And then they need to replace/remove all prefix from classes in their codebase.

**Comment by adamwathan** [comment]
Going to keep this here, not a bug so shouldn’t be an issue. I don’t think we will do anything about this ourselves any time soon I’m afraid, first time it’s been asked for in 5 years and it would be a big project. Could be cool for someone who needs it themselves to tackle as a community project though 👍🏻

**Reply by anwarulislam** [reply]
Thanks for your response

**Comment by jnovak-SM2Dev** [comment]
@adamwathan Is there an easy way to to get class names or export all class names? If there was a way to do that it would be easy to pop it in a quick js program with some regex.
